src/Console/Report/FixReport/GitlabReporter.php

```diff
@@ -52,14 +61,18 @@ public function getFormat(): string
      */
     public function generate(ReportSummary $reportSummary): string
     {
-        $about = Application::getAbout();
-
         $report = [];
         foreach ($reportSummary->getChanged() as $fileName => $change) {
             foreach ($change['appliedFixers'] as $fixerName) {
+                $description = $fixerName;
+
+                if (isset($this->fixers[$fixerName])) {
+                    $description = $this->fixers[$fixerName]?->getDefinition()?->getSummary() ?? $fixerName;
+                }
+
                 $report[] = [
                     'check_name' => 'PHP-CS-Fixer.'.$fixerName,
-                    'description' => 'PHP-CS-Fixer.'.$fixerName.' by '.$about,
+                    'description' => $description,
```

Comment:
do not remove the `about` part

Reply:
Ok I will add `about` part. 

How will the description field look like finally? Like this: ??

```
'description' => 'PHP-CS-Fixer.' . $description . ' by ' . $about,
```

Reply:
https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/9150